sdk/cosmosdb/cosmos/src/utils/supportedQueryFeaturesBuilder.ts

```diff
@@ -3,10 +3,21 @@
 import { QueryFeature } from "../common";
 
 export function supportedQueryFeaturesBuilder(disableNonStreamingOrderByQuery?: boolean): string {
-  if (disableNonStreamingOrderByQuery) {
+  const disableListAndSetAggregate = process.env.DISABLE_LIST_AND_SET_AGGREGATE === "true";
+  if (disableNonStreamingOrderByQuery && disableListAndSetAggregate) {
```

Comment:
Let's get confirmation if we need to add `disableListAndSetAggregate` flag. If yes, this could be refactored in a better way.

Reply:
Agreed, but we are planning to remove disableNonStreamingOrderByQuery, so refactoring will be necessary. It would be better to do this when we remove it or if we get any 3rd flag in the query plan.

Reply:
No, we don't need to add this flag.
https://teams.microsoft.com/l/message/19:1cf96c484c7249249455cbf15cdba3d7@thread.v2/1721751655364?context=%7B%22contextType%22%3A%22chat%22%7D